.travis.yml

```diff
@@ -23,6 +23,9 @@ matrix:
     - python: 3.7
       dist: xenial
       sudo: true
+    - python: 3.8
+      dist: xenial
+      sudo: true
```

Comment:
`dist` and `sudo` are no longer required by Travis. See #2263. You can drop them.